### PR TITLE
For issue #839 (content type vs data schema)

### PR DESCRIPTION
**Pull request by takuki**
Made the difference between content type  and data schema clearer.

***
<a href="https://pr-preview.s3.amazonaws.com/takuki/wot-thing-description/pull/847.html" title="Last updated on Nov 26, 2019, 1:58 AM UTC (5530586)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/847/2f380e7...takuki:5530586.html" title="Last updated on Nov 26, 2019, 1:58 AM UTC (5530586)">Diff</a>

### CODE REVIEW

**Comment by zolkis** [comment]
Looks good. Maybe in a  later PR we can add some examples. We can discuss about that with Daniel.
Let's not close #839 yet.

**Comment by danielpeintner** [comment]
I wonder whether the "and" to "or" change makes sense?
see https://github.com/w3c/wot-thing-description/issues/839#issuecomment-552336731

File: index.html [hunk 1]
```diff
@@ -1262,13 +1262,23 @@ <h2>Core Vocabulary Definitions</h2>
...
         (for details about semantic processing, please refer to the documentation under the namespace IRIs, e.g.,
         <a href="https://www.w3.org/2019/wot/td">https://www.w3.org/2019/wot/td</a>).
       </p>
+      <p>A data schema is an abstract notation for data contained in data formats. 
+         In a TD, concrete data formats are specified in Forms (see <a href="#form"></a>) using content types.
+         When the value of a content type in an instance of the Form is <code>application/json</code>, 
+         the data schema can be processed directly by JSON Schema processors.
+         Otherwise, Web of Things (WoT) Binding Templates [[?WOT-BINDING-TEMPLATES]] defines data schema's 
+         available mappings to other content types such as XML [[?xml]]. 
+         If the content type in an instance of the Form is not <code>application/json</code> and
```
Comment (sebastiankb) [inline index.html]: lets change "... <code>application/json</code> and ..." to "... <code>application/json</code> and if ..."

**Comment by sebastiankb** [comment]
the PR will be merged after applying the small change above

**Pull request merged**